### PR TITLE
[universal-wait-for-internet] Upgraded repo with actual wait-for-internet commands and information

### PR DESCRIPTION
**Pull request by N3rdP1um23**
Here's the PR for an earlier chat I had with Roxedus (https://github.com/linuxserver/docker-mods/pull/167). This mod will allow any docker to wait for an active internet connection before being further executed. This can be used in a case where there are pre-network requirements that need to be executed before a container has an active internet connection. As an example, waiting for an AdGuardHome docker container to finish initializing before later containers can have an active connection.

### CODE REVIEW

File: .github/workflows/BuildImage.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@ on: [push, pull_request, workflow_dispatch]
 
 env:
   ENDPOINT: "linuxserver/mods" #don't modify
-  BASEIMAGE: "replace_baseimage" #replace
-  MODNAME: "replace_modname" #replace
+  BASEIMAGE: "universal" #replace
+  MODNAME: "wait_for_internet" #replace
```
Comment (aptalca) [inline .github/workflows/BuildImage.yml:8]: these need to be dashes

Comment (N3rdP1um23) [reply .github/workflows/BuildImage.yml]: Whoops, my bad. All good now!

**Comment by Roxedus** [comment]
This should probably run at the very start of the container, delaying all other startup scripts. I suppose 15-wait-for-internet should do it.

**Comment by N3rdP1um23** [comment]
> This should probably run at the very start of the container, delaying all other startup scripts. I suppose 15-wait-for-internet should do it.

Updated numbering as suggested!

**Comment by BrodyStone21** [comment]
Sorry if this is a beginner question, but I'm up to date and have been having this issue for months. Could someone tell me how I can go about fixing it?

**Comment by aptalca** [comment]
What is the issue? This is a PR that was merged, not an issue report.

**Comment by BrodyStone21** [comment]
> What is the issue? This is a PR that was merged, not an issue report.

If SWAG starts before Pihole, I have to `nano ~/.config/swag/nginx/cf_real-ip.conf` and delete the lines that would only have a semi colon. Then, I have to restart the swag container `docker restart swag` and everything will work. Please note, I _think_ this is because of Pihole, but I could be wrong. It usually happens after rebooting my server, so that's what I assume. 

I've already cleared my logs today, but I'll try and give an example off the top of my head

```cf_real-ip.conf
;
;
123.456.789 ;
987.654.321 ;
207.123.444 ;
```

Notice how there are two lines with no ip? If I run `docker logs swag` I'll get the error `nginx: [emerg] invalid number of arguments in "set_real_ip_from" directive in /config/nginx/cf_real-ip.conf:1`

I'm here because #164 referenced here and said that this fixed it.

**Pull request merged**